<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>855</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1290</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: qlineargradient(spread:pad, x1:0.776, y1:0.614091, x2:0, y2:0.999682, stop:0.19403 rgba(82, 109, 130, 255), stop:1 rgba(31, 110, 140, 255))</string>
    </property>
    <widget class="QPushButton" name="profileInfoBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(157, 178, 191);
	border-radius: 35px;
	border : 2px solid black;
	border-color: rgb(46, 138, 153);
}

QPushButton:hover{
	background-color: rgb(132, 167, 161);
	border : 3px solid black;
	border-color: rgb(100, 204, 197)
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>profile.png</normaloff>profile.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="goHomeBtn">
     <property name="geometry">
      <rect>
       <x>1110</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(157, 178, 191);
	border-radius: 35px;
	border : 2px solid black;
	border-color: rgb(46, 138, 153);
}

QPushButton:hover{
	background-color: rgb(132, 167, 161);
	border : 3px solid black;
	border-color: rgb(100, 204, 197)
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>home.png</normaloff>home.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1197</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo app.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>1290</width>
      <height>855</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.358, y1:0.244, x2:1, y2:1, stop:0.353234 rgba(39, 55, 77, 255), stop:1 rgba(82, 109, 130, 255))
</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1290</width>
       <height>855</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1288</width>
        <height>853</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>160</y>
         <width>350</width>
         <height>400</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(70, 69, 140, 255), stop:1 rgba(27, 107, 147, 255));
border-radius: 15px;</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>80</y>
          <width>180</width>
          <height>150</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>category.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>280</y>
          <width>211</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(221, 230, 237);
</string>
        </property>
        <property name="text">
         <string>دسته بندی ها</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="gotoCategoriesBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>350</width>
          <height>400</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgba(0, 0, 0, 0);
}

QPushButton:hover{
	background-color: rgb(0, 0, 0, 50);
	border-radius: 15px;
	border : 2px solid black;
	border-color: rgb(90, 150, 227);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>160</y>
         <width>350</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(70, 69, 140, 255), stop:1 rgba(27, 107, 147, 255));
border-radius: 15px;</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>80</y>
          <width>180</width>
          <height>150</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>heart_check.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>280</y>
          <width>211</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(221, 230, 237);
</string>
        </property>
        <property name="text">
         <string>علاقه مندی ها</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="gotoFavoriteBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>350</width>
          <height>400</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgba(0, 0, 0, 0);
}

QPushButton:hover{
	background-color: rgb(0, 0, 0, 50);
	border-radius: 15px;
	border : 2px solid black;
	border-color: rgb(90, 150, 227);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>350</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(70, 69, 140, 255), stop:1 rgba(27, 107, 147, 255));
border-radius: 15px;</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>80</y>
          <width>180</width>
          <height>150</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>difference.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>280</y>
          <width>210</width>
          <height>80</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(221, 230, 237);
</string>
        </property>
        <property name="text">
         <string>مقایسه</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="gotoCompareBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>350</width>
          <height>400</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgba(0, 0, 0, 0);
}

QPushButton:hover{
	background-color: rgb(0, 0, 0, 50);
	border-radius: 15px;
	border : 2px solid black;
	border-color: rgb(90, 150, 227);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
